python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1199,6 +1254,12 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
         for v, visitSummary in enumerate(visitSummaryTables):
             visit = visitSummary[0]["visit"]
 
+            visitMap = wcsf.mapCollection.orderAtoms(f"{visit}")[0]
+            visitMapType = wcsf.mapCollection.getMapType(visitMap)
+            if (visitMap not in mapParams) and (visitMapType != "Identity"):
+                self.log.warning(f"Visit {visit} was dropped because of an insufficient number of sources.")
```

Comment:
```suggestion
                self.log.warning("Visit %d was dropped because of an insufficient number of sources.", visit)
```

Reply:
Can you explain this suggestion? Is there an issue with f-strings in logging, or are you otherwise concerned about formatting of `visit`?

Reply:
f-strings in loggers are frowned upon and non-standard where deferred formatting is the norm. For warnings it doesn't really matter because we can be very sure that the warning will always be issued (whereas for debugs you definitely don't want to use f-strings).

Reply:
What Tim said.